src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -8,64 +8,56 @@ You can pass in the same `MeetingManager` instance to multiple different `Meetin
 the `MeetingProvider`. This is to support very specific cases where you may have different React trees within you applications. For example, you are migrating
 from an Angular application to a React application.
 
-
 You can access the `MeetingManager` instance with the `useMeetingManager` hook.
 
-## Usage
-
-`MeetingProvider` must be rendered somewhere higher in the tree.
-
-```jsx
-import React from 'react';
-import { MeetingProvider, useMeetingManager } from 'amazon-chime-sdk-component-library-react';
-
-const App = () => (
-  <MeetingProvider>
-    <MyApp />
-  </MeetingProvider>
-);
-
-const MyApp = () => {
-  const meetingManager = useMeetingManager();
-
-  ...
-}
-```
-
 ## Constructor
 
-Accepts a config object of `logLevel`, `postLogConfig`, `simulcastEnabled`, `enableWebAudio`, `logger`,
-`activeSpeakerPolicy`, `videoUplinkBandwidthPolicy`, `videoDownlinkBandwidthPolicy`, `reconnectTimeoutMs`, and
-`keepLastFrameWhenPaused` that will be used when joining a meeting session. See [MeetingProvider](/docs/sdk-providers-meetingprovider--page) documentation for more details.
+Accepts a config object of `logLevel`, `postLogConfig`, `enableWebAudio`, `logger`, and `activeSpeakerPolicy` that will be used when joining a meeting session. 
+See [MeetingProvider](/docs/sdk-providers-meetingprovider--page) documentation for more details.
 
 ## Interface
 
 ### `meetingManager.join`
 
-Creates a meeting session using the meeting and attendee data passed. It will attempt to select default devices for the user.
+Creates a meeting session using the passed `MeetingSessionConfiguration`. It will attempt to select default devices for the user.
 
 ```typescript
 import { EventController } from 'amazon-chime-sdk-js';
 import { MeetingManagerConfig } from 'amazon-chime-sdk-component-library-react';
 
-(data: MeetingJoinData) => Promise<void>
-
-interface MeetingJoinData {
-  // The response of calling chime:CreateMeeting from your server.
-  meetingInfo: any;
+(meetingSessionConfiguration: MeetingSessionConfiguration, options?: MeetingManagerJoinOptions) => Promise<void>
```

Comment:
This seems to be weird. I think it should be part of section title or need a independent code block for it.
```ts
meetingManager.join(meetingSessionConfiguration: MeetingSessionConfiguration, options?: MeetingManagerJoinOptions) => Promise<void>
```